pages/reference/supervisor/device-metrics.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Device Metrics
+---
+
+# Device Metrics
+
+The {{ $names.company.lower }} Supervisor reports metrics about your device to the cloud. These metrics are available on your balenaCloud dashboard, on the device summary page:
+
+![Device Metrics](https://www.balena.io/blog/content/images/2020/09/device-metrics-1.png)
+
+You can also get device metrics data through [our open API](https://www.balena.io/docs/reference/api/overview/), which allows you to port this data to a custom dashboard to monitor the utilization and health of your fleet. Note that device metrics are available with {{ $names.os.lower }} v2.56.0+rev1 and higher, running Supervisor v11.14.0 and higher.
+
+See the following table for details about each metric reported:
+
+| Name                    | Type         | Description | API Slug | Optional? |
+|-------------------------|--------------|-------------|----------|-----------|
+| CPU utilization         | integer (%)  | Utilization percentage of the CPU. Receive updates when the "bucket" of the CPU usage changes past certain milestones in increments of 20%. | `cpu_usage` | Y |
+| CPU temperature         | integer (°C) | CPU temperature in degrees Centigrade in boundaries of 3 degrees, subject to bandwidth saving transfer strategies. | `cpu_temp` | Y |
+| Memory usage/total      | integer (Mb) | RAM usage on the device in megabytes. | `memory_usage` / `memory_total` | Y |
+| Disk storage used/total | integer (Mb) | Utilized storage and total storage in megabytes. | `storage_usage` / `storage_total` | Y |
+| CPU ID                  | string       | CPU serial number provided by manufacturer. | `cpu_id` | Y |
+| Undervoltage            | boolean      | Detect power supply insufficiency. Not supplying enough power most commonly leads to power brownouts or disk corruption. Will *not* toggle back to false after true has been reported until reboot occurs. | `is_undervolted` | N |
```

Comment:
Point them to this FAQ maybe ? https://www.balena.io/docs/faq/troubleshooting/faq/#what-to-keep-in-mind-when-choosing-power-supply-units